src/initial_conditions/initial_conditions.jl

```diff
@@ -160,6 +160,31 @@ function (initial_condition::DecayingProfile)(params)
     return local_state
 end
 
+"""
+    DYAMONDSummer(; perturb = true)
+
+An `InitialCondition` with a decaying temperature profile, and with an optional
+perturbation to the temperature.
+"""
+struct DYAMONDSummer <: InitialCondition end
+
+function (initial_condition::DYAMONDSummer)(params)
+    function local_state(local_geometry)
+        FT = eltype(params)
+        grav = CAP.grav(params)
+        thermo_params = CAP.thermodynamics_params(params)
+
+        T, p = FT(300), FT(100000) # placeholder values
```

Comment:
```suggestion
        T, p = FT(NaN), FT(NaN) # placeholder values
```

Reply:
Changing to NaN to emphasize that the calling this condition by itself is not sufficient

Reply:
Noted.